### ISSUE TITLE
socket.close is not a function

### ISSUE THREAD

**Issue by sajera** [SupremeTechnopriest/socket.io-mock#9]
Within my project i have pretty simple global mock for socket.io made by you (btw really thanks)

### jest-setup.js
```javascript

// outsource dependencies
import MockedSocket from 'socket.io-mock';
import socketIOClient from 'socket.io-client';

// NOTE mock socket.io globally
jest.mock('socket.io-client');
socketIOClient.mockReturnValue(new MockedSocket());
```

Within this project i faced up with really complex solutions which required deep integration with `socket.io` and has wrapper service for it. In order to fit all requirements was used [socket.close](https://socket.io/docs/client-api/#socket-close)

But it implementation absent within `socket.io-mock` :cry: also as `socket.disconnect`. I think it's possible to at least emulate some behavior of socket instance for client. And I am begging you to provide such functionality :innocent: Ofcourse it's possible to emulate within tests and, i do not think it will be helpful, but the solution which useful for me is:

### bloody hack
```javascript

// outsource dependencies
import MockedSocket from 'socket.io-mock';
import socketIOClient from 'socket.io-client';

// NOTE mock socket.io globally
jest.mock('socket.io-client');
MockedSocket.prototype.disconnect = jest.fn();
socketIOClient.mockReturnValue(new MockedSocket());
```

**Comment by SupremeTechnopriest** [comment]
Hey @sajera.  I will take a look at adding the functions you need today.

**Comment by SupremeTechnopriest** [comment]
@sajera Can you give this branch a go and see if it does what you need? 

https://github.com/SupremeTechnopriest/socket.io-mock/tree/disconnect

**Comment by sajera** [comment]
Greater thanks @SupremeTechnopriest yes it solve my issue with disconnect :ok_hand: . 
May i ask to implement [connected](https://socket.io/docs/client-connection-lifecycle/#Connection-status) property too ?  :sweat_smile: I know it might be not so easy :innocent:

**Comment by SupremeTechnopriest** [comment]
I think I can default connected to true and set it to false when you call disconnect.  That should be good enough.  Ill push it to the same branch.  If you can test for me that would be great.  If it works as expected ill push a new version to npm.

**Comment by sajera** [comment]
Great thanks :+1: .
It's solve the problems which i faced with :ok_hand:  and allow to remove the bloody hack from test setup.
Nicely to use such maintainable library

**Comment by SupremeTechnopriest** [comment]
I will push this to npm today.  Thanks!

**Comment by SupremeTechnopriest** [comment]
Published as `1.3.0`.  Enjoy!

**Comment by codejedi365** [comment]
Are you sure this is fixed?  I am receiving errors that `.close()` is not a function and I'm using v1.3.1.  

I am attempting to implement this library to mock socket.io connections that are established in my ReactApp upon render.  This [StackOverflow post](https://stackoverflow.com/a/61501927/3653630) is very similar to what I am trying to replicate.  If I change my App.ts to use `socket.disconnect()` rather than `socket.close()` it works but I thought disconnect is the alias determined by socket.io-client.
